_health-care/_js/_form.js

```diff
@@ -1056,37 +1095,45 @@ let xmlFieldMap = [
   { node: "SpouseNameLast",  arg: 'veteran[spouses][last_name]', transform: extractTransform },
   { node: "SpouseNameFirst",  arg: 'veteran[spouses][first_name]', transform: extractTransform },
   { node: "SpouseNameMiddle",  arg: 'veteran[spouses][middle_name]', transform: extractTransform },
-  { node: "SpouseNameSuffix",  name: 'veteran[]', arg: "JR", transform: identityTransform },
+  { node: "SpouseNameSuffix",  arg: 'veteran[spouses][suffix_name]', transform: extractTransform },
   { node: "SpouseDateOfBirthMonth",  name: 'veteran[]', arg: "4", transform: identityTransform },
   { node: "SpouseDateOfBirthDay",  name: 'veteran[]', arg: "3", transform: identityTransform },
   { node: "SpouseDateOfBirthYear",  name: 'veteran[]', arg: "1990", transform: identityTransform },
   { node: "MarraigeDateMonth",  name: 'veteran[]', arg: "6", transform: identityTransform },
   { node: "MarraigeDateDay",  name: 'veteran[]', arg: "8", transform: identityTransform },
   { node: "MarraigeDateYear",  name: 'veteran[]', arg: "2010", transform: identityTransform },
-  { node: "SpouseAddress",  name: 'veteran[]', arg: "1 Spouse Dr", transform: identityTransform },
-  { node: "SpouseCity",  name: 'veteran[]', arg: "Auburn", transform: identityTransform },
+  { node: "SpouseAddress",  arg: 'veteran[spouses][address]', transform: extractTransform },
+  { node: "SpouseCity",  arg: 'veteran[spouses][city]', transform: extractTransform },
+  //{ node: "SpouseCountry",  arg: 'veteran[spouses][country]', transform: extractTransform },
```

Comment:
Remove commented out chunks?  No reason to commit commented out code if we have revision control.